topo/topo.go

```diff
@@ -66,6 +65,8 @@ type TopologyManager interface {
 	Load(context.Context) error
 	Node(string) (node.Node, error)
 	Nodes() []node.Node
+	TopologySpecs(context.Context) ([]*topologyv1.Topology, error)
```

Comment:
i would suggest we should add clear definitions here for each method i can take a note to do this but let's at least add them here

i would suggest this interface is getting quite wide so we should look at how to improve this

Reply:
Does this look ok ? https://github.com/google/kne/pull/138/commits/6d090373e4b75f2c13c14f14af689d128e3b793e

Reply:
yep